src/Container/AddWmsPanel/AddWmsLayerEntry/AddWmsLayerEntry.spec.tsx

```diff
@@ -1,12 +1,13 @@
-import { render, screen } from '@testing-library/react';
+import { queryByAttribute, render, screen } from '@testing-library/react';
 import * as React from 'react';
 
 import OlLayerTile from 'ol/layer/Tile';
 import OlSourceTileWMS from 'ol/source/TileWMS';
 import AddWmsLayerEntry from './AddWmsLayerEntry';
 
-describe('<AddWmsLayerEntry />', () => {
+import  '../../../Util/fontawesome';
```

Comment:
I guess we should/need import this inside the component itself?

Reply:
It seems to work